tests/e2e/test_prices.py

```diff
@@ -15,59 +15,53 @@
 
 class TestPrices(unittest.TestCase):
     def setUp(self) -> None:
-        self.far_past = datetime.strptime("2022-01-01", "%Y-%m-%d")
-        # https://api.coinpaprika.com/v1/tickers/cow-cow-protocol-token/historical?start=2022-01-01&interval=1d&end=2022-04-16
-        self.first_cow_day = datetime.strptime("2022-04-15", "%Y-%m-%d")
```

Comment:
This is going to be outdated soon enough so there is no need for us to test it anymore.